module/fastgltf.cppm

```diff
@@ -0,0 +1,162 @@
+    using math::fquat;
+    using math::dquat;
+    using math::translate;
+    using math::scale;
+    using math::rotate;
+    using math::transpose;
+    using math::fmat2x2;
+    using math::fmat3x3;
+    using math::fmat4x4;
+    using math::dmat2x2;
+    using math::dmat3x3;
+    using math::dmat4x4;
+    using math::decomposeTransformMatrix;
+}
+
+namespace sources {
+    using sources::Array;
+    using sources::BufferView;
+    using sources::ByteView;
+    using sources::CustomBuffer;
+    using sources::Fallback;
+    using sources::URI;
+    using sources::Vector;
+}
+
+    using fastgltf::Accessor;
+    using fastgltf::AccessorType;
+    using fastgltf::AlphaMode;
+    using fastgltf::Animation;
+    using fastgltf::AnimationChannel;
+    using fastgltf::AnimationSampler;
+    using fastgltf::Asset;
+    using fastgltf::AssetInfo;
+    using fastgltf::Buffer;
+    using fastgltf::BufferTarget;
+    using fastgltf::BufferView;
+    using fastgltf::Camera;
+    using fastgltf::Category;
+    using fastgltf::ComponentType;
+    using fastgltf::CompressedBufferView;
+    using fastgltf::DataSource;
+    using fastgltf::Error;
+    using fastgltf::Expected;
+    using fastgltf::ExportOptions;
+    using fastgltf::Exporter;
+    using fastgltf::Extensions;
+    using fastgltf::Filter;
+    using fastgltf::GltfDataBuffer;
+    using fastgltf::Image;
+    using fastgltf::Light;
+    using fastgltf::Material;
+    using fastgltf::MaterialAnisotropy;
+    using fastgltf::MaterialSpecular;
+    using fastgltf::MaterialIridescence;
+    using fastgltf::MaterialVolume;
+    using fastgltf::MaterialTransmission;
+    using fastgltf::MaterialClearcoat;
+    using fastgltf::MaterialSheen;
+#if FASTGLTF_ENABLE_DEPRECATED_EXT
+    using fastgltf::MaterialSpecularGlossiness;
+#endif
+    using fastgltf::MaterialPackedTextures;
+    using fastgltf::Mesh;
+    using fastgltf::MimeType;
+    using fastgltf::Node;
+    using fastgltf::NormalTextureInfo;
+    using fastgltf::OcclusionTextureInfo;
+    using fastgltf::Optional;
+    using fastgltf::OptionalFlagValue;
+    using fastgltf::OptionalWithFlagValue;
+    using fastgltf::Options;
+    using fastgltf::Parser;
+    using fastgltf::PrimitiveType;
+    using fastgltf::Primitive;
+    using fastgltf::Sampler;
+    using fastgltf::Scene;
+    using fastgltf::Skin;
+    using fastgltf::SmallVector;
+    using fastgltf::span;
+    using fastgltf::SparseAccessor;
+    using fastgltf::StaticVector;
+    using fastgltf::Texture;
+    using fastgltf::TextureInfo;
+    using fastgltf::TRS;
+    using fastgltf::URI;
+    using fastgltf::URIView;
+    using fastgltf::visitor;
+    using fastgltf::Wrap;
+
+    using fastgltf::operator|;
+    using fastgltf::operator&;
+    using fastgltf::operator|=;
+    using fastgltf::operator&=;
+    using fastgltf::operator~;
+
+    using fastgltf::validate;
+    using fastgltf::getElementByteSize;
+    using fastgltf::getErrorName;
+    using fastgltf::getErrorMessage;
```

Comment:
I don't like this having any indentation. Should move this one to the left or move everything else out by one, similar to the other headers.

Reply:
Fixed.

Reply:
You missed `MappedGltfFile`, but otherwise this seems complete now, I think.

Reply:
Fixed.